store/mongo/datastore_mongo_releases.go

```diff
@@ -279,6 +280,9 @@ func (db *DataStoreMongo) GetUpdateTypes(ctx context.Context) ([]string, error)
 	}
 	var updateTypes updateType
 	err := result.Decode(&updateTypes)
+	if err == mongo.ErrNoDocuments {
+		return []string{}, nil
+	}
```

Comment:
so you want no error on no documents. ok.

Reply:
not quite... the API spec promises an array, as the endpoint is supposed to respond with an array and an empty array is a valid response if no items to populate the list exist...

What I _want_ is for the backend to validate the backend endpoints with the API specs.

Reply:
no: https://github.com/mendersoftware/deployments/blob/master/api/http/api_deployments_releases.go#L192-L196

Reply:
ok then!